src/component/page/deltakerliste-detaljer/DeltakerlisteDetaljerPage.tsx

```diff
@@ -16,40 +16,31 @@ import styles from './DeltakerlisteDetaljerPage.module.scss'
 import { KoordinatorInfo } from './KoordinatorInfo'
 import { TiltakInfo } from './TiltakInfo'
 import { KoordinatorsDeltakerliste } from '../../../api/data/deltaker'
-
-import {
-	FilterMenyVeiledere
-} from './filter-meny/FilterMenyVeiledere'
-import {
-	FilterMenyMedveileder
-} from './filter-meny/FilterMenyMedveileder'
-import { FilterMenyStatus } from './filter-meny/FilterMenyStatus'
-import { FilterMenyNavKontor } from './filter-meny/FilterMenyNavKontor'
-import { FilterMenyChips } from './filter-meny/FilterMenyChips'
+import { FilterMenyDeltakerListeDetaljer } from './filter-meny/FilterMenyDeltakerListeDetaljer'
 
 export const DeltakerlisteDetaljerPage = (): React.ReactElement => {
 	const { setTilbakeTilUrl } = useTilbakelenkeStore()
-	const params = useParams<{ deltakerlisteId: string }>()
+	const params = useParams<{deltakerlisteId: string}>()
 	const deltakerlisteId = params.deltakerlisteId || ''
 
-	useEffect(() => {
-		setTilbakeTilUrl(MINE_DELTAKERLISTER_PAGE_ROUTE)
+	useEffect( () => {
```

Comment:
nit: her også er det lagt inn mange mellomrom som skaper konflikter med det som er i main. Tenker du å endre reglene for space, eller så tenker jeg at kanskje du burde endre dine lokale instillinger